bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-control-plane.yaml

```diff
@@ -0,0 +1,268 @@
+          done
+
+          multi_network_enabled_flag=
+          if [[ "{{.OVN_MULTI_NETWORK_ENABLE}}" == "true" ]]; then
+            multi_network_enabled_flag="--enable-multi-network"
+          fi
+
+          echo "I$(date "+%m%d %H:%M:%S.%N") - ovnkube-control-plane - start ovnkube --init-cluster-manager ${K8S_NODE}"
+          exec /usr/bin/ovnkube \
+            --enable-interconnect \
+            --init-cluster-manager "${K8S_NODE}" \
+            --config-file=/run/ovnkube-config/ovnkube.conf \
+            --k8s-token-file=/var/run/secrets/hosted_cluster/token \
+            --ovn-empty-lb-events \
+            --loglevel "${OVN_KUBE_LOG_LEVEL}" \
+            --metrics-bind-address "127.0.0.1:29104" \
+            --metrics-enable-pprof \
+            --metrics-enable-config-duration \
+            ${gateway_mode_flags} \
+            --enable-multicast \
+            --disable-snat-multiple-gws \
+            --node-server-privkey ${TLS_PK} \
+            --node-server-cert ${TLS_CERT} \
+            ${multi_network_enabled_flag} \
+            --acl-logging-rate-limit "{{.OVNPolicyAuditRateLimit}}"
+        volumeMounts:
+        - mountPath: /run/ovnkube-config/
+          name: ovnkube-config
+        - mountPath: /env
+          name: env-overrides
+        - mountPath: /ovn-cert
+          name: ovn-cert
+        - mountPath: /ovn-ca
+          name: ovn-ca
+        - mountPath: /hosted-ca
+          name: hosted-ca-cert
+        - mountPath: /var/run/secrets/hosted_cluster
+          name: hosted-cluster-api-access
+        - name: ovn-master-metrics-cert
+          mountPath: /etc/pki/tls/metrics-cert
+          readOnly: True
+        resources:
+          requests:
+            cpu: 10m
+            memory: 200Mi
+        env:
+        - name: OVN_KUBE_LOG_LEVEL
+          value: "4"
+        - name: K8S_NODE
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: ALL_PROXY
+          value: socks5://127.0.0.1:8090
+        - name: NO_PROXY
+          value: kube-apiserver
+        ports:
+        - name: metrics-port
+          containerPort: 29104
+        terminationMessagePolicy: FallbackToLogsOnError
+      {{ if .HCPNodeSelector }}
+      nodeSelector:
+        {{ range $key, $value := .HCPNodeSelector }}
+        "{{$key}}": "{{$value}}"
+        {{ end }}
+      {{ end }}
+      volumes:
+      - name: ovnkube-config
+        configMap:
+          name: ovnkube-config
+      - name: konnectivity-proxy-ca
+        configMap:
+          name: konnectivity-ca-bundle
+      - name: konnectivity-proxy-cert
+        secret:
+          defaultMode: 0640
+          secretName: konnectivity-client
+      - name: env-overrides
+        configMap:
+          name: env-overrides
+          optional: true
+      - name: ovn-ca
+        configMap:
+          name: ovn-ca
+      - name: ovn-cert
+        secret:
+          secretName: ovn-cert
+      - name: ovn-master-metrics-cert
```

Comment:
this will use the ovnkube-controller-metrics-cert. I had it already in my changes, I must have messed something up between PR 1905 and this. Sorry, now it's fixed.